service/controller/v18/key/key.go

```diff
@@ -407,6 +407,10 @@ func PVCNames(customObject v1alpha1.KVMConfig) []string {
 	return names
 }
 
+func ReadinessPort(customObject v1alpha1.KVMConfig) int32 {
+	return int32(livenessPortBase + customObject.Spec.KVM.Network.Flannel.VNI)
```

Comment:
Imo if this should use the same port as the liveness probe then you should actually call LivenessPort() in here.